models/staging/ngpvan/core/stg_ngpvan__activist_codes.sql

```diff
@@ -39,4 +39,25 @@ WITH
 SELECT
     DISTINCT *
 FROM renamed
-
+QUALIFY ROW_NUMBER() OVER (
```

Comment:
Let's discuss the "multi-source dupes" in the other PR In the dbt-tmc repo but we probably want to take the same approach in all these places